chassembly/src/main/java/org/quiltmc/chasm/lang/internal/intrinsics/FlattenFunction.java

```diff
@@ -0,0 +1,41 @@
+package org.quiltmc.chasm.lang.internal.intrinsics;
+
+import org.quiltmc.chasm.lang.api.ast.ListNode;
+import org.quiltmc.chasm.lang.api.ast.Node;
+import org.quiltmc.chasm.lang.api.eval.Evaluator;
+import org.quiltmc.chasm.lang.api.exception.EvaluationException;
+
+/**
+ * The flatten intrinsic, which takes in a list of lists and returns a new list with the contents of the sublists.
+ */
+public class FlattenFunction extends IntrinsicFunction {
+    @Override
+    public Node apply(Evaluator evaluator, Node arg) {
+        if (!(arg instanceof ListNode)) {
+            throw createArgsException(arg);
+        }
+
```

Comment:
It might be useful if `flatten` could also combine lists of maps

Reply:
No longer needed because of #59